tests/test_helpers.py

```diff
@@ -115,6 +115,49 @@
      "Please rerun the helper specifying '-n list-index' to update item number 'list-index':\n"
      "1. Maria as a new contact\n"
      ),
+    (["helper", "lister", "people"],
+     "sbillinge    \nscopatz\n"),
+    (["helper", "lister", "projecta"],
+     "20ly_newprojectum    \n20sb_firstprojectum\n"),
+    (["helper", "lister", "contacts"],
+     "afriend\n"),
+    (["helper", "lister", "proposals"],
+     "20_anewproposal    \ndmref15    \nmypropsal\n"),
+    (["helper", "lister", "abstracts"],
+     "Mouginot.Model\n"),
+    (["helper", "lister", "people", "--kv_filter", "name", "simon"],
+     "sbillinge\n"),
+    (["helper", "lister", "projecta","--kv_filter", "lead", "ascopatz"],
+     "20sb_firstprojectum\n"),
+    (["helper", "lister", "contacts","--kv_filter", "institution", "columbiau"],
```

Comment:
again, you only need additional tests if it is testing new functionality, or an edge case.  e.g., for edge cases what happens if the person gives an arg that is not found at all, or is invalid in some way.  What do you want the program to do?  for the different functionality, for example pulling from a nested field (though we don't have that UC implemented yet, but it would be different functionality because it would be testing retrieving info from a nested field rather than a top-level field)

Reply:
I added a ValueError for when the key is not valid, and when the value is not found it now prints message!

Reply:
I am not sure you need value error because I think the parser always load string, but you might want to check that.

Reply:
If you give a not valid key, for example:
` rh lister people  --returrn_fields 'favorite color' 'id' `
it will return:
sbillinge       favorite color: None
dyano           favorite color: None

If the None is okay, I can just discard the ValueError. What do you think?

Reply:
I see.  I think the preferred behavior would probably be to raise a ``KeyError`` if the key is not found in that case.